espresso-server/app/src/test/java/io/appium/espressoserver/test/http/RouteDefinitionTest.java

```diff
@@ -0,0 +1,65 @@
+package io.appium.espressoserver.test.http;
+
+import fi.iki.elonen.NanoHTTPD;
+import io.appium.espressoserver.lib.handlers.RequestHandler;
+import io.appium.espressoserver.lib.handlers.exceptions.AppiumException;
+import io.appium.espressoserver.lib.model.AppiumParams;
+import io.appium.espressoserver.lib.http.RouteDefinition;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class RouteDefinitionTest {
+    private final RequestHandler<AppiumParams, Void> dumbHandler = new RequestHandler<AppiumParams, Void>() {
+        @Override
+        public Void handle(AppiumParams params) throws AppiumException {
+            return null;
+        }
+    };
+
+    private RouteDefinition routeDefinitionOne;
+    private RouteDefinition routeDefinitionTwo;
+    private RouteDefinition routeDefinitionThree;
+
+    @Before
+    public void before() {
+        routeDefinitionOne = new RouteDefinition(NanoHTTPD.Method.GET, "/hello/:param/world", dumbHandler, AppiumParams.class);
+        routeDefinitionTwo = new RouteDefinition(NanoHTTPD.Method.GET, "/hello", dumbHandler, AppiumParams.class);
+        routeDefinitionThree = new RouteDefinition(NanoHTTPD.Method.GET, "/hello/:paramOne/something/:paramTwo/world", dumbHandler, AppiumParams.class);
+    }
+
+    @Test
+    public void isMatchTest() throws Exception {
```

Comment:
I'd rather give these methods names like shouldProperlyParseRouteDefinitionsFromStrings
We anyway already have Test annotation to show it is a test